portal-api/src/utils/date.test.js

```diff
@@ -1,106 +1,146 @@
+      });
+
+      it('should return the following Monday when Friday is passed in', () => {
+        const date = fridayDate;
+
+        const result = addOneBusinessDayWithHolidays(date, holidays);
+
+        expect(result).toEqual(mondayDate);
+      });
+    });
+
+    describe('when Tuesday is a holiday', () => {
+      const mondayDate = new Date('2023-11-20');
+      const tuesdayDate = new Date('2023-11-21');
+      const wednesdayDate = new Date('2023-11-22');
+      const holidays = [tuesdayDate];
+
+      it('should return Wednesday when Monday is passed in', () => {
+        const date = mondayDate;
+
+        const result = addOneBusinessDayWithHolidays(date, holidays);
+
+        expect(result).toEqual(wednesdayDate);
+      });
+    });
+  });
+
+  describe('getFirstBusinessDayOfMonth', () => {
+    describe('when the year is 2023 and there are no holidays', () => {
+      const holidays = [];
+
+      it('should return Wednesday 1st November when the month is November', () => {
+        const dateInMonth = new Date('2023-11-10');
+
+        const result = getFirstBusinessDayOfMonth(dateInMonth, holidays);
+
+        const firstBusinessDay = new Date('2023-11-01');
+        expect(isSameDay(result, firstBusinessDay)).toBe(true);
+      });
+
+      it('should return Monday 2nd October when the month is October', () => {
+        const dateInMonth = new Date('2023-10-14');
+
+        const result = getFirstBusinessDayOfMonth(dateInMonth, holidays);
+
+        const firstBusinessDay = new Date('2023-10-02');
+        expect(isSameDay(result, firstBusinessDay)).toBe(true);
+      });
+    });
+
+    describe('when the year is 2023 and the first business day of October is a holiday', () => {
+      const firstOctoberBusinessDay = new Date('2023-10-02');
+      const holidays = [firstOctoberBusinessDay];
+
+      it('should return Monday when Thursday is passed in', () => {
+        const dateInMonth = new Date('2023-10-10');
+
+        const result = getFirstBusinessDayOfMonth(dateInMonth, holidays);
+
+        const firstBusinessDay = new Date('2023-10-03');
+        expect(isSameDay(result, firstBusinessDay)).toBe(true);
+      });
+    });
+  });
+
+  describe('getBusinessDayOfMonth', () => {
+    it('should throw an error when businessDay is less than 1', () => {
+      const dateInMonth = new Date('2023-01-01');
+      const holidays = [];
+      const businessDay = 0;
+
+      expect(() => getBusinessDayOfMonth(dateInMonth, holidays, businessDay)).toThrow(new Error('Error getting business day: business day must be a positive number'));
+    });
+
+    it('should throw an error when businessDay is not a number', () => {
+      const dateInMonth = new Date('2023-01-01');
+      const holidays = [];
+      const businessDay = 'first';
+
+      expect(() => getBusinessDayOfMonth(dateInMonth, holidays, businessDay)).toThrow(new Error('Error getting business day: business day must be a positive number'));
```

Comment:
Would prettier format these long lines?

Reply:
Yes - fixed the same thing in `getBusinessDayOfMonth.test.js`